docs/rfcs/0009-control-plane-without-k8s.md

```diff
@@ -0,0 +1,75 @@
+- Start Date: 2022-01-18
+- Target Version: 0.41.3
+
+# Summary
+
+Support install PipeCD control plane on other platform which is not k8s
+
+# Motivation
+
+Now we can deploy the control plane to kubernetes cluster, but some developers that would like to introduce PipeCD can not prepare kubernetes environments. We want to support install PipeCD control plane on platforms other than kubernetes.
+
+# Detailed design
+
+## 1. Control Plane on docker-compose
+    ### Note:
+    - Developers can deploy control plane on a single machine.
+    - Developers do not have to prepare a datastore and a filestore by themselves or they can easily use a database or a filestore on local machine.
+    ### Production Architecture
+    - Port-forward from docker-compose to your machine to enable external client access to pipecd-gateway container.
+    ![image](assets/control-plane-on-docker-compose.jpg)
+    
+    
+## 2. Control Plane on managed container services (ex. ECS)
+    ### Note:
+    - Pipecd give the abstruct architecture for ECS.
+        - Give a [sample repositpry](https://github.com/pipe-cd/control-plane-aws-ecs-terraform-demo) that describes architecure by using Terraform. 
+        - The architectures are different by each project, so create your best architecture with refering to this repository and docker-compose file.
+    - They can easily use the managed database or storage system on cloud as datastore and filestore.
+    
+    ### Production Architecture on ECS
+    ![image](assets/control-plane-on-aws.jpg)
+    
+    Note:
+    - Devide a pipecd-server and a pipecd-ops to different services because they have the same port and have different authorization.
+    - Pay attention to brocking public access to s3.
+        - Add IAM role to ECS to access S3.
+    - Make two type of target group for both of HTTP and GRPC. Use HTTPS for ALB listners because GRPC listner needs HTTPS.
+    - Set up before containers start to run so that pipecd can use config files and encryption key. You can choose from 3 options as bellow.
+        - Use Secrets Manager as config store.
```

Comment:
```suggestion
    - Set up before containers start to run so that PipeCD can use config files and encryption keys. You can choose from 3 options as below.
        - Use Secrets Manager as config store.
```